gems/pending/util/miq-process.rb

```diff
@@ -127,24 +127,7 @@ def self.processInfo(pid = nil)
   end
 
   def self.command_line(pid)
-    case Sys::Platform::IMPL
-    when :mswin, :mingw
```

Comment:
Here's some sample output from my Windows 8.1 VM, skipping over nils for background processes.

```
"taskhostex.exe "
"C:\\Windows\\Explorer.EXE"
"TPAutoConnect.exe -q -i vmware -a COM1 -F 30"
"\\??\\C:\\Windows\\system32\\conhost.exe 0x4"
"\"C:\\Program Files\\WindowsApps\\microsoft.windowscommunicationsapps_16.4.4206.722_x64__8wekyb3d8bbwe\\LiveComm.exe\" -ServerNam
e:Microsoft.WindowsLive.Platform.Server"
"C:\\Windows\\System32\\RuntimeBroker.exe -Embedding"
"\"C:\\WINDOWS\\system32\\WindowsPowerShell\\v1.0\\powershell.exe\" "
"\\??\\C:\\Windows\\system32\\conhost.exe 0x4"
"\"C:\\Program Files\\VMware\\VMware Tools\\vmtoolsd.exe\" -n vmusr"
"\"C:\\Program Files (x86)\\Common Files\\Java\\Java Update\\jusched.exe\" "
"\"C:\\Program Files (x86)\\vim\\vim74\\gvim.exe\"  ps_test.rb"
"\"C:\\Ruby22\\bin\\ruby.exe\"  .\\ps_test.rb"
```